dev_guide/environment_variables.adoc

```diff
@@ -11,75 +11,30 @@
 
 toc::[]
 
-== Overview
-
-You can set, unset or list environment variables in pods or pod
-xref:../dev_guide/templates.adoc#dev-guide-templates[templates] using the
-`oc set env` command. This command can also be used on `*BuildConfig*` objects.
-
-== CLI Interface
-
-{product-title} provides the `oc set env` command to set or unset environment
-variables for objects that have a pod template, such as replication controllers
-or deployment configurations. It can also list environment variables in pods or
-any object that has a pod template. This command can also be used on
-`*BuildConfig*` objects.
+[[using-the-oc-set-env-command]]
+== Using the oc set env Command
```

Comment:
@ahardin-rh Can I suggest a title change? "Setting and Unsetting Environment Variables"